libs/ui-lib/lib/common/components/ui/OpenShiftVersionDropdown.tsx

```diff
@@ -85,10 +85,15 @@ export const OpenShiftVersionDropdown = ({
   React.useEffect(() => {
     setCurrent(defaultLabel);
     setValue(defaultValue);
-    setHelperText(getHelperText(versions, defaultValue, t));
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [defaultLabel, defaultValue, customOpenshiftSelect]);
 
+  React.useEffect(() => {
+    if (helperTextOpenshift !== null) {
+      setHelperText(helperTextOpenshift);
```

Comment:
I dont see a reason to store this in react state or formik state. Can't you calculate the value during the rendering ? It seems that it depends on the selected ocp version which is stored in formik state.

Reply:
The problem is that  versions are different in cluster wizard page and inside the modal. Then to calculate the value it not works.